pagefaultgames/pokerogue
# [Sprite] Icon fix for Turtwig, Arceus

## What are the changes?
Replaced broken icons for turtwig and arceus with a previous one.
Curiously Arceus is does not show up bugged, but it was on the icon sheet.

@Opaque02:
https://discord.com/channels/1125469663833370665/1273029028210671748/1273995202109112360


## Why am I doing these changes?
[ [Bug] Turtwig Icon in starter-select is broken #3574 ](https://github.com/pagefaultgames/pokerogue/issues/3574)
@Opaque02 requested someone test the change and make a PR.

### Screenshots/Videos

![image](https://github.com/user-attachments/assets/724c9326-20b0-42eb-bf65-7ef35e1abcb2)
![image](https://github.com/user-attachments/assets/f87ad294-f202-4f93-beae-98153246e603)

local test:
![image](https://github.com/user-attachments/assets/e9cfc614-81fd-4c9e-9145-93a498db8d7a)

## How to test the changes?
- Enable Arceus as a starter
- Create a new game and filter starters by GenIV
- default Turtwig and default Arceus should look correct.

## Checklist
